tests/cupy_tests/core_tests/test_ndarray_elementwise_op.py

```diff
@@ -480,6 +480,12 @@ def test_add_array_boolarray(self):
     def test_iadd_array_boolarray(self):
         self.check_array_boolarray_op(operator.iadd)
 
+    def test_pos_boolarray(self):
+        for xp in (numpy, cupy):
+            a = xp.array(True, dtype=xp.bool_)
+            with pytest.deprecated_call():
+                assert a is not +a
```

Comment:
`tests/cupy_tests/core_tests/test_ndarray_unary_op.py::TestArrayUnaryOp` would be the correct place for the test.

- For `bool`, could you move the test to there?
- For the other dtypes, how about testing `a is not +a` in `test_pos_array`?

Reply:
I worried about where to place the tests when I posted this PR, thanks!